lib/imports/api/authority/server/authority-manager.js

```diff
@@ -1,8 +1,8 @@
 import { Authority } from './authority';
 import { Meteor } from 'meteor/meteor';
-import { Mongo } from 'meteor/mongo';
 import { check, Match } from 'meteor/check';
 import { DDP } from 'meteor/ddp-client';
+import { documentsColl } from './../../documents/both/collection';
```

Comment:
this isn't directly related to the issue but is a minor cleanup. I had initially passed the documents collection into constructors in case it had to be changed but that seems unnecessary so I've changed it to just be imported